### DISCUSSION TITLE
Enable inlayed error indicators shown in the Helix 23.10 news post

### DISCUSSION THREAD

**Discussion by JoshuaBlack0704** [helix-editor/helix#8653] [Q&A] [answered]
![image](https://github.com/helix-editor/helix/assets/78656867/add59ee5-d471-4df0-b92d-3ad4e9a18e87)
How do I get error indications like this? I looked through the configuration options but nothing stood out that could enable this behavior.

**Comment by kirawi** [comment, marked as answer]
Not in a release https://github.com/helix-editor/helix/pull/6417
